pkg/services/grpcserver/service.go

```diff
@@ -62,7 +62,10 @@ func ProvideService(cfg *setting.Cfg, features featuremgmt.FeatureToggles, authe
 		}
 	}
 
-	var opts []grpc.ServerOption
+	opts := []grpc.ServerOption{
+		grpc.MaxRecvMsgSize(s.cfg.GRPCServerMaxRecvMsgSize),
+		grpc.MaxSendMsgSize(s.cfg.GRPCServerMaxSendMsgSize),
+	}
```

Comment:
You can change this back to `var opts []grpc.ServerOption` as you add those in the check below anyway

Reply:
Nice catch thanks, this was a mistake. I changed my mind half way through and wanted the options added only if the configuration option is set.